kws/bin/compute_det.py

```diff
@@ -14,16 +15,17 @@
 
 import argparse
 import json
-
+from collections import defaultdict
 
 def load_label_and_score(keyword, label_file, score_file):
-    score_table = {}
+    score_table = defaultdict(list)
     with open(score_file, 'r', encoding='utf8') as fin:
         for line in fin:
             arr = line.strip().split()
             key = arr[0]
-            score = float(arr[keyword + 1])
-            score_table[key] = score
+            str_list = arr[1:]
+            scores = list(map(float, str_list))
+            score_table[key].append(scores)
```

Comment:
1. score_table.get(key, [])
2. if key not in score_table: score_table[key] = []